controllers/feeds/getFeedChat.js

```diff
@@ -214,8 +214,7 @@ const getFeedChatService = async (req, res) => {
 
     let {last_fetch_date = null} = req.query;
     if (last_fetch_date) {
-      last_fetch_date = new Date(last_fetch_date);
-      last_fetch_date = last_fetch_date.toISOString();
+      last_fetch_date = moment.utc(last_fetch_date).format('YYYY-MM-DDTHH:mm:ss');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conversion of `last_fetch_date` to UTC format using `moment.js` is a good approach for handling timezones. However, it's important to note that the `format('YYYY-MM-DDTHH:mm:ss')` method will not include milliseconds in the output string. If your application requires millisecond precision, you should use `format()` without any arguments to get the full ISO 8601 date string including milliseconds and 'Z' to indicate UTC.
```diff
- last_fetch_date = moment.utc(last_fetch_date).format('YYYY-MM-DDTHH:mm:ss');
+ last_fetch_date = moment.utc(last_fetch_date).format();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->